GUI/Properties/Resources.de-DE.resx

```diff
@@ -0,0 +1,310 @@
+  <data name="MainRepoContacting" xml:space="preserve"><value>Kontaktiere das Repository...</value></data>
+  <data name="MainRepoUpdating" xml:space="preserve"><value>Aktualisiere Repository...</value></data>
+  <data name="MainRepoFailedToConnect" xml:space="preserve"><value>Die Verbindung zum Repository konnte nicht hergestellt werden: {0}</value></data>
+  <data name="MainRepoUpToDate" xml:space="preserve"><value>Repository bereits aktuell.</value></data>
+  <data name="MainRepoFailed" xml:space="preserve"><value>Aktualisierung des Repositorys fehlgeschlagen!</value></data>
+  <data name="MainRepoSuccess" xml:space="preserve"><value>Repository erfolgreich aktualisiert.</value></data>
+  <data name="MainRepoAutoRefreshPrompt" xml:space="preserve"><value>Möchtest du, dass CKAN die Modliste jedes Mal mit dem Repository abgleicht und aktualisiert, wenn sie geladen wird? (Du kannst sie jederzeit auch mit der Schaltfläche in der Menüleiste aktualisieren.)</value></data>
+  <data name="MainRepoBalloonTipDetails" xml:space="preserve"><value>{0} Aktualisierung(en) verfügbar</value></data>
+  <data name="MainRepoBalloonTipTooltip" xml:space="preserve"><value>Zum Aktualisieren hier klicken</value></data>
+  <data name="MainTrayIconResume" xml:space="preserve"><value>Fortsetzen</value></data>
+  <data name="MainTrayIconPause" xml:space="preserve"><value>Anhalten</value></data>
+  <data name="MainTrayNoUpdates" xml:space="preserve"><value>Keine Aktualisierungen verfügbar</value></data>
+  <data name="MainTrayUpdatesAvailable" xml:space="preserve"><value>{0} Aktualisierung(en) verfügbar</value></data>
+  <data name="MainWaitPleaseWait" xml:space="preserve"><value>Bitte warten</value></data>
+  <data name="MainWaitDone" xml:space="preserve"><value>Alles erledigt!</value></data>
+  <data name="ManageKspInstancesNotValid" xml:space="preserve"><value>"{0}" ist kein gültiges KSP-Verzeichnis.</value></data>
+  <data name="NewRepoDialogFailed" xml:space="preserve"><value>Die Master-Liste konnte nicht abgerufen werden.</value></data>
+  <data name="SettingsDialogSummmary" xml:space="preserve"><value>{0} Dateien, {1}</value></data>
+  <data name="SettingsDialogSummaryInvalid" xml:space="preserve"><value>Ungültiger Pfad: "{0}"</value></data>
+  <data name="SettingsDialogCacheDescrip" xml:space="preserve"><value>Wähle einen Ordner um CKANs Mod-Downloads zu speichern:</value></data>
+  <data name="SettingsDialogDeleteConfirm" xml:space="preserve"><value>Möchten Sie wirklich {0} Dateien aus dem Cache löschen und damit {1} freigeben?</value></data>
+  <data name="AddAuthTokenTitle" xml:space="preserve"><value>Authentifikationstoken hinzufügen</value></data>
+  <data name="AddAuthTokenToken" xml:space="preserve"><value>Token:</value></data>
+  <data name="AddAuthTokenAccept" xml:space="preserve"><value>&amp;Annehmen</value></data>
+  <data name="AddAuthTokenCancel" xml:space="preserve"><value>&amp;Abbrechen</value></data>
```

Comment:
These two actions will conflict if you press `A` in the dialog

Reply:
@HebaruSan can you move the `&` to the `c` of `Abbrechen`, so `Abbre&amp;chen`?
`C` makes sense too, that's next to `Q` a shortcut often used for quitting/canceling stuff.

Reply:
Fixed. Thanks for catching that1